src/transformers/models/detr/configuration_detr.py

```diff
@@ -155,6 +155,10 @@ def __init__(
         bbox_loss_coefficient=5,
         giou_loss_coefficient=2,
         eos_coefficient=0.1,
+        in_chans=3,
+        pretrained=True,
+        freeze_layers=True,
+        fix_batch_norm=True,
```

Comment:
What's the reason you want to replace the frozen batch norm layers?

Reply:
Training from scratch, I want to try training from fully randomly initialised model. Also I don't have pretrained backbone for my problem anyway, so I think this parameter won't harm